docs/service/processor.rst

```diff
@@ -98,7 +98,7 @@ You can also use it with ``fetch_crashids``:
 
 .. code-block:: shell
 
-   app@socorro:/app$ socorro-cmd fetch_crashids --num=1 | xargs scripts/process_crashes.sh
+   app@socorro:/app$ ./socorro-cmd fetch_crashids --num=1 | ./scripts/process_crashes.sh
```

Comment:
I'm pretty sure `.` is not on the `PATH`, so we always have to do `./socorro-cmd`. If that's not true, that's cool and we can remove them all from this file. But it felt weird to have half this way and half that way.

Reply:
It's in my path:

```
$ make shell
app@19f44b639b99:/app$ echo $PATH
/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:.
```

That appears to be set in ``/etc/bash.bashrc``, but that also seems to set the prompt, which doesn't work for me 🤷‍♂️

Reply:
It's probably overridden by `/home/app/.bashrc`. So, building the container adds the `/etc/bash.basrhc`, but then a few months later, I needed to do something that required a home directory so I changed adduser to build a home directory and while I was puzzled why "socorro" didn't appear in the prompt, I never looked into it until now. That's exciting!

Reply:
We could probably have `docker/set_up_ubuntu.sh` delete `/home/app/.bashrc`. Then we get the right prompt and path for all users.

Reply:
Oh, I know.... the problem I have is doing this:

```
$ docker-compose run app shell socorro-cmd
```

That won't find socorro-cmd. But doing this will:

```
$ docker-compose run app shell ./socorro-cmd
```

That's really irritating.